numpy/ma/extras.py

```diff
@@ -258,9 +253,9 @@ def getdoc(self):
         doc = getattr(npfunc, '__doc__', None)
         if doc:
             sig = self.__name__ + ma.get_object_signature(npfunc)
-            locdoc = "Notes\n-----\nThe function is applied to both the _data"\
-                     " and the _mask, if any."
-            return '\n'.join((sig, doc, locdoc))
+            doc = ma.doc_note(doc, "The function is applied to both the _data "
```

Comment:
I wonder if as a follow-up we can set `self.__wrapped__ = npfunc` to avoid having to add a manual signature line.